src/SectionTitle.jsx

```diff
@@ -34,6 +33,12 @@ class SectionTitle extends React.Component {
 					</Chunk>
 				</FlexItem>
 				{children &&
+					<Chunk>
+						<FlexItem shrink
+							>
```

Comment:
this doesn't really line up with anything. I think it should just go on the line above